src/components/ProgressBar/ProgressBar.props.ts

```diff
@@ -0,0 +1,14 @@
+export interface IProgressBarProps {
+    title: string;
+    width: number;
+    height: number;
+    percentage: string;
```

Comment:
@hbagarich percentage is currently only used for display but we could have either only percentage or total and used values, but for me total and used seems like better option as it is more flexible.